README.md

```diff
@@ -116,6 +116,7 @@ Pull app will automatically watch and pull in upstream's default (master) branch
         conflictReviewers:      # Optional, on merge conflict assign a reviewer
           - wei
     label: ":arrow_heading_down: pull"  # Optional
+    conflictLabel: "merge-conflict"     # Optional, on merge conflict assign a label
```

Comment:
We always assign a label on merge conflict and there's no way to turn it off right? This comment may give an impression that this label may be optional. What do you think?

Reply:
Yes, this comment should be fine I think
`# Optional, on merge conflict assign a custom label, Default : merge-conflict`